dev/tests/unit/Magento/FunctionalTestFramework/Console/BaseGenerateCommandTest.php

```diff
@@ -15,11 +14,6 @@
 
 class BaseGenerateCommandTest extends TestCase
 {
-    public function tearDown(): void
-    {
-        AspectMock::clean();
-    }
-
```

Comment:
As a precaution, can we set tests and suites to [] for`TestObjectHandler` and `SuiteObjectHandler` in tearDown()?

Reply:
@jilu1 I changed tearDown method and set [] for TestObjectHandler and SuiteObjectHandler